h/services/__init__.py

```diff
@@ -59,6 +59,9 @@ def includeme(config):
     )
     config.register_service_factory(".links.links_factory", name="links")
     config.register_service_factory(".group_list.group_list_factory", name="group_list")
+    config.register_service_factory(
+        ".group_scope.group_scope_factory", name="group_scope"
```

Comment:
It would have been easier to follow if this commit came before the one that uses the `group_scope` service in the `group_list` service.

Reply:
Yeah, but it _does_ come before it...in my local branch! GH doesn't respect re-ordering commits during rebases on PR branches, apparently? https://help.github.com/en/articles/why-are-my-commits-in-the-wrong-order